crates/bins/src/bin/datadog-static-analyzer.rs

```diff
@@ -251,18 +251,28 @@ fn main() -> Result<()> {
         // if there is no config file, we must read the rules from a file.
         // Otherwise, we exit.
```

Comment:
We don't exit now, so the comment may be Invalid

Reply:
updated, thank you!